Makefile

```diff
@@ -1,4 +1,5 @@
-PYTHON?=python
+SHELL := /bin/bash
+PYTHON?=/usr/bin/env python3
```

Comment:
At least in Ubuntu 20.04, by default python3 is installed to `/usr/bin/python3` and `/usr/bin/python` is unset. Since this seems to be a python3 compatible project, I added this line.